modules/registry-operator-config-resources-secret-gcp.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * registry/configuring-registry-storage-gcp-user-infrastructure.adoc
+
+[id="registry-operator-config-resources-secret-gcp_{context}"]
+= Configuring a Secret for the Image Registry Operator
+
+In addition to the `configs.imageregistry.operator.openshift.io` and ConfigMap
+resources, configuration is provided to the Operator by a separate Secret
+resource located within the `openshift-image-registry` namespace.
+
+The `image-registry-private-configuration-user` Secret provides
+credentials needed for storage access and management. It overrides the default
+credentials used by the Operator, if default credentials were found.
+
+For GCS on GCP storage it is expected to contain one key whose value is the
+contents of a credentials file provided by GCP:
+
+* REGISTRY_STORAGE_GCS_KEYFILE
+
+.Procedure
+
+* Create an {product-title} Secret that contains the required keys.
++
+[source,subs="attributes+"]
+----
```

Comment:
Do we need  line here, like:
_For storage other than GCS on GCP, run the following command

And below
_For GCS on GCP storage, run the following command_